client/my-sites/stats/site.jsx

```diff
@@ -58,7 +58,7 @@ import StatsModuleSearch from './features/modules/stats-search';
 import StatsModuleTopPosts from './features/modules/stats-top-posts';
 import StatsModuleUTM, { StatsModuleUTMOverlay } from './features/modules/stats-utm';
 import StatsModuleVideos from './features/modules/stats-videos';
-import StatsFeedbackController from './feedback';
+import StatsFeedbackCard from './feedback';
```

Comment:
This needs to call the `StatsFeedbackController` top-level component now. I'll fix this one up.